src/components/Meeting.tsx

```diff
@@ -119,6 +120,13 @@ export function Meeting({ meeting, searchWords, tags }: MeetingProps) {
             ))}
           </Box>
         )}
+        <Box>
+          <Report 
+            meetingId={meeting.meetingId} 
+            meetingName={meeting.name}
+            meetingEmail={meeting.meetingEmail}          
```

Comment:
maybe just pass the whole `meeting`?

Reply:
Working on this - I now pass just pass the whole meeting - when I console.log(meeting) in the change data function I do have all the meeting props available but they are not passing through to the email anymore.  Will keep you updated